yktrace/src/swt.rs

```diff
@@ -121,36 +122,68 @@ unsafe fn yk_swt_start_tracing_impl() {
     }
 
     TRACE_BUF_CAP = TL_TRACE_INIT_CAP;
-    TRACING = true;
+    __YK_SWT_ACTIVE = true;
 }
 
 /// Record a location into the trace buffer if tracing is enabled on the current thread.
 #[do_not_trace]
 #[no_mangle]
+#[cfg(not(all(target_arch = "x86_64", target_os = "linux")))]
```

Comment:
Let's add comments with a few words about the "fast path" and "slow path" functions.